### PR TITLE
Add form intro field to action types.

### PR DESCRIPTION
**Pull request by katzmo**
See https://github.com/moreonion/campaignion_starterkit/pull/105

### CODE REVIEW

File: modules/campaignion_donation_type/campaignion_donation_type.features.field_instance.inc [hunk 1]
```diff
@@ -111,6 +111,60 @@ function campaignion_donation_type_field_default_field_instances() {
...
+    'bundle' => 'donation',
+    'custom_add_another' => '',
+    'custom_remove' => '',
+    'default_value' => array(
+      0 => array(
+        'value' => '',
+      ),
+    ),
+    'deleted' => 0,
+    'description' => '',
```
Comment (torotil) [inline modules/campaignion_donation_type/campaignion_donation_type.features.field_instance.inc:125]: Should we add a description that states where/how this is displayed?

**Review by torotil — APPROVED** [review]
Perfect. Only raising the question of whether we would want to have a description for the field.

File: modules/campaignion_donation_type/campaignion_donation_type.features.field_instance.inc (hunk 1 above)
Comment (katzmo) [reply modules/campaignion_donation_type/campaignion_donation_type.features.field_instance.inc:125]: I tried but it looks forlorn in ae-admin, not obvious which field it belongs to. I’d hope the title is clear enough?

Comment (torotil) [reply modules/campaignion_donation_type/campaignion_donation_type.features.field_instance.inc:125]: Ok, then good as is. ;-)

**Pull request merged**